pipeline/trusted-resources/README.md

```diff
@@ -0,0 +1,35 @@
+# Pipelines In Pipelines
+
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/tektoncd/experimental/blob/master/LICENSE)
+
+This is an experimental project to provide a seperate webhook for remote resources verification.
```

Comment:
Can you elaborate on this? Is there a doc or issue I can read to catch up on the goal of this project? It sounds very interesting 👍

Reply:
Yes! This is proposed by @wlynch. The idea is to allow users to install/deploy a webhook in current tekton pipeline, where we can verify the remote resources (taskrun, pipelinerun, oci bundle etc.) with Cosign. 
Maybe he can share the doc with you?

Reply:
super rough notes here: https://hackmd.io/93mfJPyDQKCyn0IKwjzgWQ#Design-Details - tl;dr it's basically a modification of Jason's original trusted task TEP, but using validation webhooks so we can prototype something before needing to make modifications to the Pipelines API.